services/QuillLMS/app/controllers/api/v1/active_activity_sessions_controller.rb

```diff
@@ -32,6 +32,16 @@ def destroy
     render(plain: 'OK')
   end
 
+  private def working_params
+    return valid_params unless params.dig(:active_activity_session, :passage)
+    permitted_passage_object = params[:active_activity_session][:passage].map do |paragraph| 
```

Comment:
Is this going to error if there's no [:passage]? I think only Proofreader has that. Might be worth testing this code for a different activity type.

Reply:
@emilia-friedberg This should be addressed by the guard as well. Thanks for raising though